object_test.go

```diff
@@ -665,64 +665,156 @@ func TestObject_InList(t *testing.T) {
 }
 
 func TestObject_ContainsKey(t *testing.T) {
-	reporter := newMockReporter(t)
-
-	value := NewObject(reporter, map[string]interface{}{"foo": 123, "bar": ""})
-
-	value.ContainsKey("foo")
-	value.chain.assert(t, success)
-	value.chain.clear()
-
-	value.NotContainsKey("foo")
-	value.chain.assert(t, failure)
-	value.chain.clear()
-
-	value.ContainsKey("bar")
-	value.chain.assert(t, success)
-	value.chain.clear()
-
-	value.NotContainsKey("bar")
-	value.chain.assert(t, failure)
-	value.chain.clear()
-
-	value.ContainsKey("BAR")
-	value.chain.assert(t, failure)
-	value.chain.clear()
+	testObj := map[string]interface{}{"foo": 123, "bar": ""}
```

Comment:
I think tests will be more readable if we inline (copy-paste) testObj into each test. This way you can quickly understand the test by just looking at the table entry.

We could do it for TestObject_ContainsKey and TestObject_ContainsValue - where testObj is small.